packages/metrics/src/features/registration/index.ts

```diff
@@ -22,6 +25,12 @@ export interface IDeathRegistrationFields extends IPoint {
   deathDays: number | undefined
 }
 
+export interface IUserAuditFields extends IPoint {
+  data: string | undefined | Record<string, any>
```

Comment:
Can we really get a `Record<string, any>` from the database? If I understand this correctly, this is used as the type for the data coming from InfluxDB measurement right?

Reply:
i think it's for sending data into the database. You suggested it in a previous comment.  I will rather we keep it simple as a string. Let me remove it